pkg/api/validation/validation.go

```diff
@@ -1760,6 +1760,9 @@ func ValidateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 		if mountpoints.Has(mnt.MountPath) {
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must be unique"))
 		}
+		if !path.IsAbs(mnt.MountPath) {
```

Comment:
what happens today if you use a non-absolute path? is it relative to the container root?